jobwatch.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, task, filename,
         self.exclude_errors = exclude_errors
         self.requires = requires
         self.maxage = maxage
+        self.type = None
```

Comment:
Is this a Py3 fix or a previously-existing bug or?  I don't see it used in the changes.

Reply:
Actually this is gone now. At some point I got an AttributeError with that `type`, but it got fixed somewhere along the way.